newsfragments/3636.other

```diff
@@ -0,0 +1 @@
+Tahoe-LAFS manual's table of contents page has been reorganized.
```

Comment:
This could be a `.documentation` fragment now, right?

Reply:
Renamed it in 3d9ccde.  The PR that added `.documentation` was added in a PR that started after this.